Example/PDFExporterDemo/PDFExporterDemo/Controllers/SettingsScreen/SettingsViewController.h

```diff
@@ -20,7 +20,7 @@
 
 @property (weak, nonatomic) id<SettingsViewControllerDelegate> delegate;
 
-@property (nonatomic) UIEdgeInsets contentPaperInsets;
+@property (nonatomic) UIEdgeInsets contentPaperInset;
```

Comment:
could be a typo here, as the type is UIEdgeInsets and the name of the property is Inset

Reply:
As per Apple naming convention, we the singular (e.g.: contentInset from `UIScrollView`, separatorInset from `UITableViewCell`).